src/Host/Client/Impl/DataInspection/REnvironments.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.R.DataInspection {
+    /// <summary>
+    /// Expressions for various common R environments, suitable for use as <c>environmentExpression</c> argument
+    /// when calling <see cref="RSessionExtensions.EvaluateAndDescribeAsync"/>.
+    /// </summary>
+    public static class REnvironments {
```

Comment:
`Definitions` maybe?

---
In reply to: [63222890](https://github.com/Microsoft/RTVS/pull/1770#discussion_r63222890) [](ancestors = 63222890)